app/_src/gateway-operator/get-started/konnect/deploy-data-plane.md

```diff
@@ -67,19 +74,90 @@ To get the endpoint and the authentication details of the data plane.
               volumeMounts:
                 - name: cluster-certificate
                   mountPath: /var/cluster-certificate
-                - name: kong-cluster-cert
-                  mountPath: /etc/secrets/kong-cluster-cert/
+                - name: konnect-client-tls
+                  mountPath: /etc/secrets/konnect-client-tls/
                   readOnly: true
             volumes:
               - name: cluster-certificate
-              - name: kong-cluster-cert
+              - name: konnect-client-tls
                 secret:
-                  secretName: kong-cluster-cert
+                  secretName: konnect-client-tls
                   defaultMode: 420
     ' | kubectl apply -f -
     ```
-    The results should look like this:
+
+    The result should look like this:
 
     ```text
     dataplane.gateway-operator.konghq.com/dataplane-example configured
     ```
+
+{% endif_version %}
+
+{% if_version gt:1.3.x %}
+
+1. Extract the following values from the **Configuration parameters** step 4:
+   1. `CP_ID`: find the value of `cluster_server_name`. The first segment of that value is the control plane ID for your cluster. For example, if the value of `cluster_server_name` is `36fc5d01be.us.cp0.konghq.com`, then the control plane ID of the cluster is `36fc5d01be`
+   1. `REGION`:  find the value in the bottom left corner of the screen.
+   1. `HOSTNAME`:  the server you are connected to (e.g. `konghq.tech`, `konghq.com`).
+
+1. Replace the values above in the following manifest and deploy it with `kubectl apply`
+
+   ```yaml
+    echo '
+    kind: KonnectExtension
+    apiVersion: gateway-operator.konghq.com/v1alpha1
+    metadata:
+      name: example-konnect-config
+      namespace: kong
+    spec:
+      controlPlaneRef:
+        type: konnectID
+        konnectID: <CP_ID>
+      controlPlaneRegion: <REGION>
+      serverHostname: <HOSTNAME>
+      konnectControlPlaneAPIAuthConfiguration:
+        clusterCertificateSecretRef:
+          name: konnect-client-tls
+    ' | kubectl apply -f -
+    ```
+
+    The result should look like this:
+
+    ```text
+    konnectextension.gateway-operator.konghq.com/example-konnect-config created
+    ```
+
+1. Deploy your data plane that references such a `KonnectExtension` with `kubectl apply`:
```

Comment:
```suggestion
1. Deploy your data plane and reference the `KonnectExtension` you just configured with `kubectl apply`:
```

Reply:
The DataPlane in the manifest references the extension, so no action is required from the user.